shell/browser/api/atom_api_web_contents.cc

```diff
@@ -458,6 +468,23 @@ void WebContents::InitWithSessionAndOptions(
   prefs->subpixel_rendering = params->subpixel_rendering;
 #endif
 
+#if defined(OS_MACOSX)
+  base::TimeDelta interval;
+  if (ui::TextInsertionCaretBlinkPeriod(&interval))
+    prefs->caret_blink_interval = interval;
```

Comment:
500msec would be a good fallback default here too

Reply:
macOS automatically falls back to 500 :)